### PR TITLE
Add two custom metrics for video using source elements with media queries

### PR DESCRIPTION
**Pull request by scottjehl**
The first metric returns the number of video elements that have a source element using media. The second is an array of all video source media attribute values in play.

These metrics are meant to keep track of responsive video usage now that the features are standard and supported again.

---

**Test websites**:

- https://scottjehl.com/posts/using-responsive-video/
- https://scottjehl.com/sandbox/video-media/

### CODE REVIEW

**Comment by rviscomi** [comment]
@scottjehl could you add a few test websites to your first comment, matching the format in the [PR template](https://github.com/HTTPArchive/custom-metrics/blob/main/.github/pull_request_template.md?plain=1)? The bot will run the custom metric changes against those sites and spit out the results so we can make sure that it works.

**Comment by scottjehl** [comment]
Sure thing @rviscomi , I added a couple links to the PR in that format

File: dist/media.js [hunk 1]
```diff
@@ -123,6 +123,12 @@ return JSON.stringify({
     'video_display_style' : Array.from(document.querySelectorAll('video')).map(video => {
       return getComputedStyle(video, null).getPropertyValue('display');
     }),
+    //Returns the number of video elements that contain a source element that uses a media attribute
+    'video_using_source_media_count': document.querySelectorAll('video:has(source[media])').length,
+    //Returns the media attribute values in use on video source elements
+    'video_source_media_values': Array.from(document.querySelectorAll('video source[media]')).map(source => {
+        return source.getAttribute('media')
```
Comment (rviscomi) [inline dist/media.js:130]: Making a no-op change to trigger the WPT bot to rerun the tests
```suggestion
        return source.getAttribute('media');
```

**Review by rviscomi — APPROVED** [review]
Looks like it works!

**Pull request merged**